packages/ui/src/theme/Color.ts

```diff
@@ -165,10 +165,488 @@ export enum Color {
   Yellow500 = '#B84807',
 }
 
+export enum ColorLightInternal {
+  //
+  // ???
+  //
+
+  White = '#fff',
+  White50 = 'rgba(255, 255, 255, 0.5)',
+  White40 = 'rgba(255, 255, 255, 0.4)',
+  White20 = 'rgba(255, 255, 255, 0.2)',
+  White10 = 'rgba(255, 255, 255, 0.1)',
+  White08 = 'rgba(255, 255, 255, 0.08)',
+  Transparent = 'rgba(0, 0, 0, 0)',
+
+  Black = '#000',
+  Black50 = 'rgba(0, 0, 0, 0.5)',
+  Black20 = 'rgba(0, 0, 0, 0.2)',
+
+  //Colors with 30% opacity
+  Dark30 = 'rgba(143, 148, 158, 0.3)',
+  Blue30 = 'rgba(0, 112, 245, 0.3)',
+  Green30 = 'rgba(3, 135, 47, 0.3)',
+  Red30 = 'rgba(229, 35, 13, 0.3)',
+  Silver30 = 'rgba(225, 229, 234, 0.3)',
+
+  //Colors with 10% opacity
+  Blue10 = 'rgba(0, 112, 245, 0.1)',
+  Red10 = 'rgba(195, 25, 9, 0.1)',
+
+  //
+  // Legacy Neutral Colors
+  //
+
+  /**@deprecated Use Dark100 */
+  Grey100 = '#8F949E',
+  /**@deprecated Use Dark200 */
```

Comment:
@MekhrangizY @doniyor2109 I think we replaced deprecated colors in the design system, maybe we can get rid of them in V2?